packages/shared-react/src/useInterval.ts

```diff
@@ -18,8 +18,7 @@ along with the library. If not, see <http://www.gnu.org/licenses/>.
 
```

Comment:
Hm, is it necessary? We don't need it yet...

Reply:
I like having the proper structure early, but I'm ok with changing later.

Reply:
I am in favor of updating the structure only when needed. So, since it's a nit, I'd say we leave it as it is for now :) We can create a "components" folder when we have a component to put inside.